meson.build

```diff
@@ -0,0 +1,5 @@
+project('pgroonga', 'c', version: '4.0.2')
```

Comment:
We should set `license`: https://mesonbuild.com/Reference-manual_functions.html#project_license